assembl/views/api2/discussion.py

```diff
@@ -540,6 +540,13 @@ def extract_taxonomy_csv(request):
             message = "no message"
         if not message:
             message = "no message"
+
+        if thematic == "no thematic associated":
+            idea_ids = m.Idea.get_idea_ids_showing_post(query.id)
+            for thematic_id in reversed(idea_ids):
+                if db.query(m.Idea).filter(m.Idea.id == thematic_id).first().title:
+                    thematic = db.query(m.Idea).filter(m.Idea.id == thematic_id).first().title.best_lang(user_prefs).value
```

Comment:
rename variable to `thematic_title` and I agree that you should add a variable for the thematic